functions/task2.js

```diff
@@ -0,0 +1,10 @@
+function validate(age) {
```

Comment:
name `validate` does not show that this function should return boolean value. Use another name like `calculateIsAdult`, `getIsAdult` etc